docs/contrib/contribute.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Contributing to Taskwarrior
+---
+
+Help is needed in all areas of Taskwarrior development - design, coding, testing, support and marketing.
+Applicants must be friendly.
+Perhaps you are looking to help, but don't know where to start.
+You can of course [email us](mailto:taskwarrior-dev@googlegroups.com) but take a look at this list.
+Perhaps you have skills we are looking for, here are ways you may be able to help:
+
+-   Use Taskwarrior, become familiar with it, and make suggestions.
+    We get great feedback from both new users and veteran users.
+    New users have a fresh approach that we can no longer achieve, while veteran users develop clever and crafty ways to use the product.
+
+-   Report bugs and odd behavior when you see it.
+    We don't necessarily know it's broken, unless you tell us.
+
+-   Suggest enhancements.
+    We get lots of these, and it's great.
+    Some really good ideas have been suggested and implemented.
+    Sure, some are out of scope, or plain crazy, but the stream of suggestions is fascinating to think about.
+
+-   Participate in the [bug tracking](https://github.com/GothenburgBitFactory/taskwarrior/issues) database, to help others and maybe learn something yourself.
+
+-   Help [triage](/docs/triage) the issues list.
+
+-   Join the IRC channel \#taskwarrior on freenode.net and help answer some questions.
```

Comment:
freenode is dead, long live libera.chat (👉🏻 https://web.libera.chat/#taskwarrior)